### ISSUE TITLE
Translations for the Serbo-Croatian languages

### ISSUE THREAD

**Issue by elsandosgrande** [ContributorCovenant/contributor_covenant#732]
While Bosnian, my native language, is already present, it is only present in Latin, while both Latin AND Cyrillic are official scripts in Bosnia. Also, there are no Croatian and Serbian variants.

As an example of what I mean, look at the final state of https://github.com/InBetweenNames/gentooLTO/pull/434 . As soon as you give me the green light, I will start working on the translations to Serbian and Croatian and the transcription to Cyrillic for Bosnian.

And no, I do not know of any other speakers of the languages in question who would be able to review the translations (not counting my mother, as she does not have a GitHub account; she is an English professor by the way). The person who did the Bosnian translation should suffice for all of the translations, as all of the languages in question are mutually intelligible, but I can understand if you wish for people born in the respective countries to review the translations. 

I eagerly await for your response.

**Comment by CoralineAda** [comment]
That sounds great, thank you in advance!

**Comment by elsandosgrande** [comment]
Oh, also, "saradnici" is "coworkers", not "contributors". "doprinositelji" and "pridonositelji" are though. This is from the second sentence of the translated code of conduct. My mother does note that "saradnici" sounds more natural/is more common that the latter two, but I still think that the latter two are more correct. Then again, a lot of things regarding git sound unnatural once I translate them, or make an attempt at least, so...

Edit
===

So... Am I to start work on the translations right now, or...

Edit two
===

Should we cc the original author of the translation, at least for the sake of politeness? Judging from https://github.com/ContributorCovenant/contributor_covenant/blob/release/content/version/1/4/code-of-conduct.bs.md, it's https://github.com/fjallarefur.

**Comment by triggerNZ** [comment]
I'm happy to review the croatian one if birth in the country is a requirement.

**Comment by elsandosgrande** [comment]
@triggerNZ Honestly, I am more concerned for the Bosnian one, since I generally tend to lean towards Croatian terminology, like saying "tisuća" instead of "hiljada", "e-pošta" instead of "imejl" and so on. Of course, I double-check with Bing and Google translators and then run that by my mother.

#### Side note
@CoralineAda The formatting for the original (English) covenant is very strange. I mean, there can be three line breaks in one sentence at times, which just looks to me like whoever wrote it was not able to turn "Word wrap" on in Vim or their editor of choice.

**Comment by triggerNZ** [comment]
@elsandosgrande yeah i i can't help too much with the bosnian one either, I'd just be defaulting to Croatian.

**Comment by elsandosgrande** [comment]
It's no biggie. As long as one of them passes with flying colors, the others will almost certainly follow suit.

Now, what's taking Coraline so long?

**Comment by CoralineAda** [comment]
Taking me so long for what? I'm not clear on what you need from me?

**Comment by elsandosgrande** [comment]
To quote the two edits from my comment following yours: "

Edit
===

So... Am I to start work on the translations right now, or...

Edit two
===

Should we cc the original author of the translation, at least for the sake of politeness? Judging from https://github.com/ContributorCovenant/contributor_covenant/blob/release/content/version/1/4/code-of-conduct.bs.md, it's https://github.com/fjallarefur."

**Comment by CoralineAda** [comment]
Yes, that sounds like a good plan! :)

**Comment by elsandosgrande** [comment]
CC-ing him, retouching the translation, or both?

**Comment by CoralineAda** [comment]
Both. They may want the opportunity to review or otherwise participate.

**Comment by elsandosgrande** [comment]
I thank thee for your swift reply!

cc @fjallarefur

**Comment by illfygli** [comment]
@elsandosgrande I am not the original author of the translation. I converted the website to use a static site generator, and it seem that git now thinks that I authored a lot of things.
It looks like @ivana-mcconnell contributed the original translation.

**Comment by elsandosgrande** [comment]
Oh... All right then. That was unexpected.

**Comment by elsandosgrande** [comment]
Three years and no reaction to the mention. What would be the best course of action?
